pandas/indexes/category.py

```diff
@@ -136,6 +136,19 @@ def _simple_new(cls, values, name=None, categories=None, ordered=None,
         result._reset_identity()
         return result
 
+    @Appender(_index_shared_docs['_shallow_copy'])
+    def _shallow_copy(self, values=None, categories=None, ordered=None,
+                      **kwargs):
```

Comment:
@jreback Not now, because ``_simple_new`` is a class method.

```
# on current master
pd.CategoricalIndex([1, 2, 3], ordered=True)._simple_new([1, 2])
# CategoricalIndex([1, 2], categories=[1, 2], ordered=False, dtype='category')
```